modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -130,10 +139,33 @@ const getOptions = (port: number, browserName: string, settings: DriverSettings,
     }
   }
 
+  // Remote webdriver settings
+  if (settings.remoteWebdriver) {
+
+    caps.browserVersion = 'latest';
+
+    if (browserName == 'firefox') {
```

Comment:
These things are not obvious why are they needed, what do they do?

Reply:
Comments have been added to the code. In this case, we added better logging for firefox for readability purposes